package.json

```diff
@@ -60,6 +60,8 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^1.8.0",
     "loader-utils": "~0.2.16",
+    "npm-run-all": "3.1.1",
```

Comment:
Do I really need npm-run-all ? You don't seem to use it, and it's probably a dependency of npm-submodules if your lib needs it ?